src/Database/Migrator.php

```diff
@@ -228,15 +240,19 @@ public function getMigrationFiles($path)
     }
 
     /**
-     * Load a migration instance from a file.
+     * Resolve a migration instance from a file.
      *
-     * @param  string $file
      * @param  string $path
-     * @return object
+     * @param  string $file
+     * @return array
      */
-    public function resolve($file, $path)
+    public function resolve($path, $file)
     {
-        return $this->files->getRequire("$path/$file.php");
+        $migration = $path . '/' . $file . '.php';
```

Comment:
Nope. Interpolation, not injection. I was suggesting to change it back to what I had. ;)

~~~php
$migration = "$path/$file.php";
~~~